.github/workflows/wait-ofborg.yml

```diff
@@ -7,9 +7,11 @@ jobs:
     steps:
     - name: Wait for ofborg CI
       run: |
-        # wait for ~30min...
+        if [[ "$BASE_BRANCH" == "refs/heads/staging" ]]; then
```

Comment:
Please add a comment why this is necessary.

Reply:
Added a comment.